Containerfile.bpfman.openshift

```diff
@@ -0,0 +1,52 @@
+# We do not use --platform feature to auto fill this ARG because of incompatibility between podman and docker
+ARG BUILDPLATFORM=linux/amd64
+
+FROM --platform=$BUILDPLATFORM registry.access.redhat.com/ubi9/ubi-minimal:9.4-1194 AS bpfman-build
+ARG DNF_CMD="microdnf"
+
+ARG BUILDPLATFORM
+
+# The following ARGs are set internally by docker/build-push-action in github actions
+ARG TARGETARCH=amd64
+ARG TARGETOS=linux
+ARG TARGETPLATFORM=linux/amd64
+
+RUN echo "TARGETOS=${TARGETOS}  TARGETARCH=${TARGETARCH}  BUILDPLATFORM=${BUILDPLATFORM}  TARGETPLATFORM=${TARGETPLATFORM}"
+
+WORKDIR /usr/src/bpfman
+COPY ./ /usr/src/bpfman
+
+RUN ${DNF_CMD} update -y \
+    && ${DNF_CMD} install -y rust cargo openssl-devel \
```

Comment:
We're  still bundling a version of the fiat-crypto crate.
This crate contains implementations of elliptic-curve cryptography that is *NOT* approved to be shipped by Fedora/RH *in any form* (i.e. also not as source code).
We will need to patch out any references to the p434 curve *before* compressing the vendor tarball.
We can take the patch from the Fedora package for the crate (rust-fiat-crypto).